### PR TITLE
Upload assets

### PR DESCRIPTION
**Pull request by paulolieuthier**

### CODE REVIEW

**Review by tonidas — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    uses: actions/github-script@v5
+    env:
+      AWS_ACCESS_KEY_ID: ${{ steps.aws-credentials.outputs.aws-access-key-id }}
+      AWS_SECRET_ACCESS_KEY: ${{ steps.aws-credentials.outputs.aws-secret-access-key }}
+      AWS_SESSION_TOKEN: ${{ steps.aws-credentials.outputs.aws-session-token }}
+      ASSETS_NAME: ${{ inputs.name }}
+      ASSETS_PATH: ${{ inputs.path }}
+      ASSETS_BASE_PATH: assets/${{ steps.aws-credentials.outputs.subject }}/run:${{ env.GITHUB_RUN_ID }}/attempt:${{ env.GITHUB_RUN_ATTEMPT }}
+    with:
+      script: |
```
Comment (t0rr3sp3dr0) [inline action.yml]: This is essentially calling bash from node, we should use run instead.

**Review by t0rr3sp3dr0 — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (paulolieuthier) [reply action.yml]: The reason I chose Javascript was to use the globber auxiliary module.

Comment (paulolieuthier) [reply action.yml]: But we can indeed use github-script to generate the file list and then execute the rest in a bash action.

**Comment by paulolieuthier** [comment]
@t0rr3sp3dr0 reimplemented in bash script.

**Comment by paulolieuthier** [comment]
@t0rr3sp3dr0 I missed the push.

**Pull request merged**